packages/swingset-liveslots/src/vatDataTypes.d.ts

```diff
@@ -105,7 +104,9 @@ export type DefineKindOptions<C> = {
    * `prepareExoClass` use this internally to protect their raw class methods
    * using the provided interface.
    */
-  interfaceGuard?: InterfaceGuard<unknown>;
+  interfaceGuard?:
+    | InterfaceGuard<unknown>
+    | Record<string, InterfaceGuard<unknown>>;
```

Comment:
Can it really be either value or is this being loose with the cases?

IIUC it's a single `InterfaceGuard` unless it's a part of an Exo "kit" in which cases it's the Record. 
https://github.com/Agoric/agoric-sdk/blob/b9b86253042124e54a30c118f1eb39aa73f41a5a/packages/vat-data/src/exo-utils.js#L195-L200

WDYT of making that type more explicit? e.g. `interfaceGuard?: InterfaceGuard` and `interfaceGuards?: Record<string, InterfaceGuard<unknown>>`. Then in https://github.com/Agoric/agoric-sdk/blob/b9b86253042124e54a30c118f1eb39aa73f41a5a/packages/swingset-liveslots/src/virtualObjectManager.js#L951-L968 it `interfaceGuards` would be used in the multifaceted case.

Without this, the `defineKind` options are misleading to the developer.

Reply:
Done, so resolving.